test/Serilog.Settings.Configuration.Tests/Support/ClassHierarchy.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Serilog.Settings.Configuration.Tests.Support
+{
+    public abstract class DummyAbstractClass
```

Comment:
I think this class (and its subclasses) are no longer needed. If this is the case, the file can probably be removed :)